tests/unit/test_crypto/test_keypairs_bls.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2022 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Tests for BLS KeyPair module of the Crypto Package."""
+from cosmpy.crypto.keypairs_bls import (
+    PrivateKey,
+    PublicKey,
+    aggregate_signatures,
+    verify_aggregated_signatures,
+)
+
+
+def test_basic_usecase():
+    """Test common BLS keys and signatures use case."""
+    msg = b"some"
+    pk1 = PrivateKey()
+    signature1 = pk1.sign(msg)
+    assert pk1.verify(msg, signature1)
```

Comment:
Can we use a hard-coded representation of the ground-truth of the signatures for these expectations, rather than generating it from the library which we're testing? We could then assert that the generated signature matches this ground truth in addition.

Reply:
will check it

Reply:
done